tests/testProcessCcd.py

```diff
@@ -58,6 +58,7 @@ def setUp(self):
                      "isr.assembleCcd.setGain=False",
                      # This test uses CP-MasterCal calibration products
                      "-C", "%s/processCcdCpIsr.py" % configPath]
+        argsList.append('--doraise')
```

Comment:
Can you clarify what this line is adding? Was testProcessCcd failing silently before in some situations?